src/components/IconDetail.vue

```diff
@@ -1,14 +1,24 @@
 <template>
   <div class="p-2 flex flex-col md:flex-row md:text-left relative">
-    <IconButton class="absolute top-0 right-0 p-3 text-2xl flex-none leading-none" icon="carbon:close" @click="$emit('close')" />
-    <div :style="{color: previewColor}">
+    <IconButton
+      class="absolute top-0 right-0 p-3 text-2xl flex-none leading-none"
+      icon="carbon:close"
+      @click="$emit('close')"
+    />
+    <div :style="{ color: previewColor }">
       <ColorPicker v-model:value="previewColor" class="inline-block">
         <Icon class="p-4 text-8xl" :icon="icon" />
       </ColorPicker>
     </div>
     <div class="px-6 py-2 mb-2 md:px-2 md:py-4">
       <button
-        class="text-gray-500 hover:text-primary text-sm dark:text-dark-500 !outline-none"
+        class="
+          text-gray-500
+          hover:text-primary
+          text-sm
+          dark:text-dark-500
+          !outline-none
+        "
```

Comment:
Did you change any lines in this file? Could you revert the format changes to keep this PR simple? Thanks.

Reply:
Sorry about that mess, my IDE auto formats on save. I have reverted the format changes, should be clean now.